AnomalyDetection-AFD/README.md

```diff
@@ -0,0 +1 @@
+This code is to be purely used as a reference  to the accompanying video blog. It's not meant to compile without changes discussed in the video.
```

Comment:
## Reference code for Azure WAF blog and Video for Process Alerts in Azure FrontDoor WAF

This code is to be purely used as a reference to the accompanying video blog. It's not meant to compile without changes discussed in the video.

## Contributing

This project welcomes contributions and suggestions.  Most contributions require you to agree to a
Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.

When you submit a pull request, a CLA bot will automatically determine whether you need to provide
a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
provided by the bot. You will only need to do this once across all repos using our CLA.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.